go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/onsi/gomega v1.27.6
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.14.0
-	github.com/submariner-io/admiral v0.15.0-m4.0.20230405145112-f36881ade1a3
+	github.com/submariner-io/admiral v0.15.0-m4.0.20230407143241-c46968df8a7a
```

Comment:
This will need to be updated to the commit used for https://github.com/submariner-io/admiral/pull/577 after it merges.